### DISCUSSION TITLE
Как переопределить t-content?

### DISCUSSION THREAD

**Discussion by omqmo** [taiga-family/taiga-ui#5510] [Q&A] [answered]
Как переопределить .t-content (нужно изменить padding) у tui-accordion-item?

**Comment by waterplea** [comment, marked as answer]
Придётся подлезть во внутреннюю реализации через ::ng-deep. В будущем планируется редизайн аккордеона с больше свободой и простотой кастомизации.

**Reply by omqmo** [reply]
Остается надеятся, что будущие @taiga-ui наступит раньше будущего, касающегося ::ng-deep, angular.
